test.ts

```diff
@@ -29,8 +29,10 @@ export const runContainer = async (
   return containerID.trim();
 };
 
-// executeScriptInContainer finds the only "coder_script"
-// resource in the given state and runs it in a container.
+/**
+ * Finds the only "coder_script" resource in the given state and runs it in a
+ * container.
+ */
```

Comment:
Updated the comment formatting so that these would be public, and would show up when someone hovers over the function definition

Reply:
TIL: `/**` are public comments and `//` are private 🤦